recipes/binutils/all/conanfile.py

```diff
@@ -188,13 +195,20 @@ def package_info(self):
         self.user_info.prefix = self.options.prefix
         self.output.info(f"executable prefix={self.options.prefix}")
 
+        # v2 exports
+        self.buildenv_info.append_path("PATH", bindir)
+        self.buildenv_info.append_path("PATH", absolute_target_bindir)
+        self.conf_info.define("user.binutils:gnu_triplet", self.options.target_triplet)
+        self.conf_info.define("user.binutils:prefix", self.options.prefix)
+
         # Add recipe path to enable running the self test in the test package.
         # Don't use this property in production code. It's unsupported.
         self.user_info.recipe_path = os.path.realpath(__file__)
+        self.conf_info.define("user.binutils:recipe_path",  os.path.realpath(__file__))
```

Comment:
Honestly, is it really used in any downstream recipe? Maybe user_info stuff have not been translated to conf because it was useless?